lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -26,3 +26,8 @@ export type AccountInfo = {
     idTokenClaims?: TokenClaims & { [key: string]: string | number | string[] | object | undefined | unknown };
     nativeAccountId?: string;
 };
+
+export type ActiveAccountHomeLocalIds = {
```

Comment:
```suggestion
export type ActiveAccountFilters = {
```

Reply:
Something a bit more generic in case we want to add additional filter properties in the future

Reply:
Don't see these changes, have they been pushed?

Reply:
Pushed